src/Microsoft.Azure.SignalR.Emulator/HubEmulator/DynamicHubContext.cs

```diff
@@ -24,10 +24,14 @@ internal class DynamicHubContext
             ConnectionHandler = connectionHandler;
         }
 
+        public DynamicHubContext()
```

Comment:
Remove if not used

Reply:
Oh I need this for unit tests, since I changed property `UserGroupManager` to virtual and its required parameterless constructor

Reply:
What about making it `protected`